collect_app/src/main/java/org/odk/collect/android/map/MapConfigurator.java

```diff
@@ -34,31 +37,57 @@ public Option(String id, int sourceLabelId, BaseLayerSource source) {
 
     /** Defines the available base layer sources, in their order of appearance. */
     private static Option[] initOptions() {
-        TileSourceFactory factory = new TileSourceFactory(Collect.getInstance().getApplicationContext());
         return new Option[] {
             new Option("google", R.string.base_layer_source_google,
                 new GoogleBaseLayerSource()),
             new Option("mapbox", R.string.base_layer_source_mapbox,
                 new MapboxBaseLayerSource()),
             new Option("osm", R.string.base_layer_source_osm,
-                new WmsBaseLayerSource(org.osmdroid.tileprovider.tilesource.TileSourceFactory.MAPNIK)
+                new WmsBaseLayerSource(
+                    new WebMapService(
+                        "Mapnik", 0, 19, 256, OSM_COPYRIGHT,
+                        "http://a.tile.openstreetmap.org/{z}/{x}/{y}.png",
+                        "http://b.tile.openstreetmap.org/{z}/{x}/{y}.png",
+                        "http://c.tile.openstreetmap.org/{z}/{x}/{y}.png"
+                    )
+                )
             ),
             new Option("usgs", R.string.base_layer_source_usgs,
                 new WmsBaseLayerSource(
                     GeneralKeys.KEY_USGS_MAP_STYLE, R.string.usgs_map_style,
-                    new WmsOption("topo", R.string.usgs_map_style_topo, factory.getUSGSTopo()),
-                    new WmsOption("hybrid", R.string.usgs_map_style_hybrid, factory.getUsgsSat()),
-                    new WmsOption("imagery", R.string.usgs_map_style_imagery, factory.getUsgsImg())
+                    new WmsOption("topo", R.string.usgs_map_style_topo, new WebMapService(
```

Comment:
The applyBundle() interface gets the serialized information about each Web Map Service in its Bundle, so there's no need for the odk.collect TileSourceFactory anymore.

(This also has the nice side effect that it doesn't confusingly duplicate the name of the other TileSourceFactory class in osmdroid.)  Oddly, our TileSourceFactory in odk.collect used to create tile sources for USGS that were identical to those in osmdroid _except_ they omitted the copyright notice. The behaviour here with WebMapService should be completely identical, except that I've restored those copyright strings.